platform/platform.go

```diff
@@ -24,17 +24,32 @@ type Platform interface {
 	Regrade([]RegradeSpec) error
 }
 
+type RemotePlatform interface {
+	Platform
+	Ping() error
```

Comment:
(I am in two minds about this, but I did end up folding Ping into the Platform interface.)